Dockerfile

```diff
@@ -58,17 +40,28 @@ RUN export DEBIAN_FRONTEND='noninteractive' && \
       /usr/bin/python python /usr/bin/python3 100 && \
     curl -sSf https://sh.rustup.rs | sh -s -- -y && \
     cargo install sccache --features=gcs && \
+    BASE_URL=https://dl.google.com/android/repository/ && \
+    curl -sSf \
+      -L ${BASE_URL}/commandlinetools-linux-${ANDROID_SDK_VERSION}_latest.zip \
+      -o /tmp/android-sdk.zip && \
+    unzip /tmp/android-sdk.zip -d /tmp && \
+    mkdir -p ${ANDROID_SDK_ROOT}/cmdline-tools/latest && \
+    mv /tmp/cmdline-tools/* ${ANDROID_SDK_ROOT}/cmdline-tools/latest && \
     yes | sdkmanager --licenses && \
+    sdkmanager --update && \
     sdkmanager \
-      "platforms;android-22" \
+      'platforms;android-22' \
       "platforms;android-${ANDROID_SDK_PLATFORM_VERSION}" \
-      "build-tools;${ANDROID_SDK_BUILD_TOOLS_VERSION}" \ 
+      "build-tools;${ANDROID_SDK_BUILD_TOOLS_VERSION}" \
       platform-tools \
       ndk-bundle && \
+    sdkmanager --uninstall emulator && \
```

Comment:
This saves us around 700 MB in size. Since the emulators aren't used they can be removed.

Reply:
neat! I wish we should just NOT install them in the first place, but oh well.